testsuite/pytests/test_get_set.py

```diff
@@ -41,6 +41,35 @@
     HAVE_PANDAS = False
 
 
+@nest.ll_api.check_stack
+class TestNestGetSet(unittest.TestCase):
+    """nest module get/set tests"""
+
+    def setUp(self):
+        nest.ResetKernel()
+
+    def test_get(self):
+        # TestCase.setUp calls ResetKernel so kernel attributes should be equal to their
```

Comment:
Please turn it into a docstring for the test.

Reply:
It's specific to what just the first lines do, I added a docstring for the test but left this as a comment.